src/test_workflow/integ_test/distribution_zip.py

```diff
@@ -0,0 +1,43 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+import os
+import subprocess
+
+from system.zip_file import ZipFile
+from test_workflow.integ_test.distribution import Distribution
+
+
+class DistributionZip(Distribution):
+    def __init__(self, filename: str, version: str, work_dir: str) -> None:
+        super().__init__(filename, version, work_dir)
+
+    @property
+    def install_dir(self) -> str:
+        return os.path.join(self.work_dir, f"{self.filename}-{self.version}")
+
+    @property
+    def config_dir(self) -> str:
+        return os.path.join(self.install_dir, "config")
+
+    def install(self, bundle_name: str) -> None:
+        logging.info(f"Installing {bundle_name} in {self.install_dir}")
+        with ZipFile(bundle_name, "r") as zip:
+            zip.extractall(self.work_dir)
+
+    @property
```

Comment:
The right way to do this is inheritance: create `DistributionZipOpenSearch` and `DistributionZipOpenSearchDashboards` and pick which instance to create in a `create` class method. Then the first one would implement `start_cmd` returning one filename and the second the other.

Reply:
Since the scope of this is way bigger than adding windows integTest support.